### PR TITLE
Nav header feature

### PR DESCRIPTION
**Pull request by fobadara**
#nav-header-feature 

In this version I:

* created a navigation-bar and homepage in my index.html file
* styled them in my style.css file
* added necessary images through my images folder

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,347 @@
+@media screen and (max-width: 400px) {
+  body {
+    all: unset;
+    background-color: #e5e5e5;
```
Comment (Shaher-11) [inline style.css]: You need to change the background color to the one used in the original design

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <!-- If section or article is used here linter error displays missing head tag hence the use of div -->
+        <div class="main-container">
+            <nav class="navigation" id="top-nav">
+                <div class="paragraph" id="welcome">Welcome</div>
+                <div id="hamburger"><img src="images/ham.svg" alt=""></div>
+            </nav>
+            <header class="header-container" id="top-header">
+                <div class="header-container" id="header-1-container">
+                    <h1 class="header header-1" id="hey">Hey there. I'm Abdulfatai. </h1>
+                    <h1 class="header header-1" id="job">I'm a software developer</h1>
```
Comment (Shaher-11) [inline index.html:20]: Change the text color as shown in the original design as well as h1 tag
#### Original design
right

#### Your Project
wrong

File: index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Fobadara's Portfolio</title>
+    </head>
+    <body>
+        <!-- If section or article is used here linter error displays missing head tag hence the use of div -->
+        <div class="main-container">
+            <nav class="navigation" id="top-nav">
+                <div class="paragraph" id="welcome">Welcome</div>
+                <div id="hamburger"><img src="images/ham.svg" alt=""></div>
```
Comment (Shaher-11) [inline index.html]: Increase the size of the welcome word and menu icon

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @fobadara,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: style.css (hunk 1 above)
Comment (fobadara) [reply style.css]: Oh, thank you mistook the background color for font color on figma

File: index.html (hunk 2 above)
Comment (fobadara) [reply index.html:20]: Ok

File: index.html (hunk 3 above)
Comment (fobadara) [reply index.html]: I am done with all the corrections

File: style.css (hunk 1 above)
Comment (rloterh) [reply style.css]: Well resolved

File: README.md [hunk 4]
```diff
@@ -1 +1,69 @@
-# my-portfolio-site-2
\ No newline at end of file
+# my-portfolio-site-1
+
```
Comment (rloterh) [inline README.md:2]: - [ ] OPTIONAL 
Your ReadMe file appears not very well organized and very descriptive. Please add a more descriptive README file to your project by using [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# .gitignore
+#files here are ignored
```
Comment (rloterh) [inline .gitignore:2]: - [ ] OPTIONAL
Your image directory contains two large bin files as can be seen below. Kindly include these files in the gitignore file.
image

image

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html:20]: Not resolved

File: index.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Fobadara's Portfolio</title>
+    </head>
+    <body>
+        <!-- If section or article is used here linter error displays missing head tag hence the use of div -->
+        <div class="main-container">
```
Comment (rloterh) [inline index.html:12]: - [ ] As requested by the previous reviewer, kindly make sure that your design matches the provided [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) as much as possible. Kindly ensure that your design matches the provided colors, for background, alignments: margins, and paddings.

Your design:
image

The original design:
image

File: index.html [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+            <nav class="navigation" id="top-nav">
```
Comment (rloterh) [inline index.html]: - [ ] OPTIONAL
As specified kindly ensure that the section is enclosed with a `header` tag

File: index.html [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+                <div class="paragraph" id="welcome">Welcome</div>
+                <div id="hamburger"><img src="images/ham.svg" alt=""></div>
+            </nav>
+            <header class="header-container" id="top-header">
```
Comment (rloterh) [inline index.html]: - [ ] OPTIONAL
As specified kindly ensure that this section is enclosed with a `section` tag

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Fobadara, 

You did great on this project, and for making the changes requested by the previous reviewer!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 Kindly consider my OPTIONAL comments.

## Good Points 👍  
- No linter errors

## Required Changes ♻️
- [ ] _Check the comments under the review._


## Optional suggestions
_Please keep in mind that you, for best practices and following the project requirements and the design templates closely kindly consider these aspects as outlined in the below comments._

**_Kindly notice that these are required changes that I'm sorry they didn't show up in the earlier review._**



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (fobadara) [reply index.html:20]: I am really impressed by your review. You actually took the time to tell what I am doing wrong and how to go about it. Thank you.

File: index.html (hunk 6 above)
Comment (vichuge) [reply index.html:12]: - [ ] It looks like you change the background color as requested, but you forgot the margins and paddings, if you observe the original, this has some spaces between the end screen and the info in `<nav>`, according to an image that's the request for my last partner. I'm pretty sure you can improve this a little bit more. 🐶 
image

Comment (vichuge) [reply index.html:12]: - [ ] It looks like you change the background color as requested, but you forgot the margins and paddings, if you observe the original, this has some spaces between the end screen and the info in `<nav>`, according to an image that's the request for my last partner. I'm pretty sure you can improve this a little bit more. 🐶 
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi dear student, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

star_wars_almost_there


## Aspects to improve ♻️
_Check the comments under the review._

- [ ] Check the comments under the review. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SpaYco** [comment]
Hello 👋,

_This is an additional review as a result of the second opinion request._

You're right, this simple change shouldn't block you from continuing the program, but I **highly recommend** that you do it! these small details can help you in the future when job hunting!

As this was the only reason that your project was not approved it will be approved as soon as possible.

Keep rocking 🚀!

**Comment by fobadara** [comment]
Thanks so much. You are a life saver. I will ensure I implement those
changes

**Comment by nidalaa** [comment]
Hello @fobadara 👋 


The status of your project in the Dashboard should have changed and you should be able to submit that project now. In case of any troubles please open a Zendesk ticket.

**Comment by fobadara** [comment]
Thank you, it has changed.

**Pull request merged**